components/Image/index.tsx

```diff
@@ -76,7 +79,7 @@ const Image = (Props: ImageProps) => {
               alt={alt}
               loading={lazy ? 'lazy' : 'eager'}
             />
-            <figcaption>{caption}</figcaption>
+            <figcaption>{children || caption}</figcaption>
```

Comment:
I'm imaging a ticket in the future that gets immediately put in the backlog.

Reply:
hahaha shhh